platform/action/core/platform-action-core-impl/pom.xml

```diff
@@ -27,6 +27,18 @@
         <dependency>
             <groupId>ddf.action.core</groupId>
             <artifactId>action-core-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-api</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.fastjson</groupId>
+            <artifactId>boon</artifactId>
+            <version>0.30</version>
```

Comment:
https://github.com/codice/ddf/blob/master/pom.xml#L154

Reply:
changed to `${boon.version}`